awx/main/models/projects.py

```diff
@@ -284,15 +284,9 @@ class Meta:
         help_text=_('Allow changing the SCM branch or revision in a job template ' 'that uses this project.'),
     )
 
-    # indicate if content signature validation is enabled
-    signature_validation = models.BooleanField(
-        default=False,
-        help_text=_('Enable signature validation.'),
-    )
     # credential (keys) used to validate content signature
     signature_validation_credential = models.ForeignKey(
         'Credential',
-        related_name='%(class)ss_signature_validation',
```

Comment:
i guess this was not correct before and should still be
`related_name='%(class)ss`